### PR TITLE
What's new 8.10

### LINKED ISSUE #3851: What's new in 8.10
Please add your features and enhancements for `8.10`. Don't forget to include the related PR link! 

### Rules Management 

- [x] **New prebuilt rule details flyout**:  The new prebuilt rule details flyout allows you to examine the details of a prebuilt rule before you install it, or before you update it. You can access this flyout by clicking a rule name on either the **Add Elastic Rules** page or the **Rule updates** table. The flyout displays the About, Definition, and Schedule sections, as shown on the rule details page. It also shows the setup and investigation guides for rules that have them. 
https://github.com/elastic/security-docs/pull/3850
   

### Detection Engine 

- [x] **Custom highlighted fields**: You can now specify additional highlighted fields for personalized alert investigation flows. Fields with data are added to the << investigation-section,Highlighted fields >> section within the alert details flyout. You can also find custom highlighted fields in the About section of the rule details page.

- [ ] **MITRE ATT&CK® coverage page**: The **MITRE ATT&CK® coverage** page shows which [MITRE ATT&CK®](https://attack.mitre.org) adversary tactics and techniques are covered by your installed and enabled detection rules. This includes both Elastic prebuilt rules and custom rules. (https://github.com/elastic/security-docs/pull/3881)

### Explore

- [x] **Updates in the Security navigation menu**: The Security navigation menu has been updated with reorganized sections and a refreshed design. In addition, a new Rules section has been added, from which you can access the following pages:
- Rules
- Benchmark Integrations
- Shared Exception Lists
- MITRE ATT&CK® coverage
https://github.com/elastic/security-docs/pull/3785
https://github.com/elastic/security-docs/pull/3884

### Investigations

- [ ] **Redesigned alert details flyout** - Summary pending

## Generative AI

- [x] **Role-based access control for AI Assistant**
A new RBAC setting controls user access to the AI Assistant. 
https://github.com/elastic/security-docs/pull/3809

## Defend Workflows/Asset Management 

- [x] **Reputation service**: When configuring malicious behavior protections on an Elastic Defend policy, you can now select to use Reputation service. This service identifies malicious activity and false positives, and enriches alerts using data from various sources, such as VirusTotal and telemetry. For example, reputation service can detect suspicious downloads of binaries with low or malicious reputation.
NOTE: Reputation service requires an active [Platinum or Enterprise subscription](https://www.elastic.co/pricing) and is available on cloud deployments only.
https://github.com/elastic/security-docs/pull/3855 

## Cloud Security 

- [x] **Organization-wide onboarding for cloud security posture management on AWS**
This release automates the onboarding of every account within an AWS Organization to cloud security posture management (CSPM) — including existing and new accounts. With AWS CloudFormation, onboarding takes just a few clicks. This helps you quickly get a comprehensive view of the security posture of all your AWS accounts — both current and future.
https://github.com/elastic/security-docs/pull/3806

- [x] **Cloud security posture management, now for Google Cloud**
Elastic Security 8.10 expands its cloud security posture management (CSPM) capabilities to cover Google Cloud. You can now assess and bolster the security posture of your GCP assets right from our platform. 
https://github.com/elastic/security-docs/pull/3806

## Endpoint 

- [ ] Add features here. 

## Protections Experience

- [ ] Add features here. 

## ResponseOps

- [ ] No changes to Cases in 8.10.

### PR DESCRIPTION
**Pull request by natasha-moore-elastic**
- Resolves #3851 

Preview: [What’s new in 8.10](https://security-docs_3901.docs-preview.app.elstc.co/guide/en/security/master/whats-new.html)

### CODE REVIEW

File: docs/whats-new.asciidoc [hunk 1]
```diff
@@ -4,131 +4,93 @@
...
-
-* You now have the option to {security-guide}/cases-open-manage.html#cases-ui-open[specify a category] for new and existing cases.
-* You can now {security-guide}/cases-open-manage.html#cases-lens-visualization[add Lens visualizations to cases] from anywhere within the {security-app}.
-* The case details *Alerts* tab now displays the number of alerts attached to a case.
-* Email notifications now follow a new and improved template.
+=== Cloud security posture management, now for Google Cloud
 
+{elastic-sec} 8.10 expands its cloud security posture management (CSPM) capabilities to cover Google Cloud. You can now assess and bolster the security posture of your GCP assets right from our platform.
 
+// link to be added when PR #3809 is merged {security-guide}/cspm-get-started-gcp.html.html[Google Cloud].
```
Comment (natasha-moore-elastic) [inline docs/whats-new.asciidoc:94]:
```suggestion
// link to be added when PR #3806 is merged {security-guide}/cspm-get-started-gcp.html.html[Google Cloud].
```

File: docs/whats-new.asciidoc [hunk 2]
```diff
@@ -4,131 +4,93 @@
...
+The new prebuilt rule details flyout allows you to examine the details of a prebuilt rule before you {security-guide}/prebuilt-rules-management.html[install or update] it. You can access this flyout by clicking a rule name on either the **Add Elastic Rules** page or the **Rule updates** table. The flyout displays the About, Definition, and Schedule sections, as shown on the rule details page. It also shows the setup and investigation guides for rules that have them.
+ 
 [role="screenshot"]
-image::whats-new/images/8.9/IG-UI.png[Interactive investigation guide]
-
+image::whats-new/images/8.10/prebuilt-rule-details-flyout.png[Prebuilt rule details flyout]
 
 [float]
-=== Prebuilt rule updates
+=== Redesigned alert details flyout
```
Comment (nastasha-solomon) [inline docs/whats-new.asciidoc:55]:
```suggestion
=== Enhanced alert details flyout UI
```

File: docs/whats-new.asciidoc [hunk 3]
```diff
@@ -4,131 +4,93 @@
...
 
-Check out the {security-guide}/prebuilt-rules-downloadable-updates.html[latest updates] to prebuilt rules. To download the latest updates, refer to {security-guide}/prebuilt-rules-management.html#update-prebuilt-rules[Update Elastic prebuilt rules]. 
+To be added
```
Comment (nastasha-solomon) [inline docs/whats-new.asciidoc:57]:
```suggestion
The redesigned alert details flyout gives you more options and flexibility when investigating an alert. Check the sections in the right panel to get a summary of important alert details. Analyze each section further by opening their detailed views. Additional core improvements include:

* Previews of rule details and visualizations allow you to stay within the flyout when investigating the alert.
* Investigation guides are easier to find and read.
* Alert insights now include prevalence information on related hosts and users.

```

Comment (nastasha-solomon) [reply docs/whats-new.asciidoc:57]: @jmikell821 you can reuse the [gif](https://www.elastic.co/guide/en/security/master/images/open-alert-details-flyout.gif) I created for the alert details docs. The file is `open-alert-details-flyout.gif` and it's located at: `docs/detections/images/open-alert-details-flyout.gif`

Comment (nastasha-solomon) [reply docs/whats-new.asciidoc:57]: Changes @paulewing suggested via Slack:

```suggestion
The redesigned alert details experience presents relevant context, insights, and more while investigating an alert. Utilize the collapsed view to see summarized information and then expand each section further to open detailed views. Additional improvements include:

* Previews of rule details and visualizations allow you to stay within the flyout when investigating the alert.
* Investigation guides are easier to find and read.
* Alert insights now include prevalence information on related hosts and users.
```

File: docs/whats-new.asciidoc [hunk 4]
```diff
@@ -4,131 +4,91 @@
...
 
 [float]
-=== Prebuilt rule updates
-
-Check out the {security-guide}/prebuilt-rules-downloadable-updates.html[latest updates] to prebuilt rules. To download the latest updates, refer to {security-guide}/prebuilt-rules-management.html#update-prebuilt-rules[Update Elastic prebuilt rules]. 
+=== Enhanced alert details flyout UI
 
-[float]
-=== Manage and filter alert tags
+The redesigned alert details experience presents relevant context, insights, and more, while investigating an alert. Use the collapsed view to access summarized information, and then expand each section further to open detailed views. Additional improvements include:
```
Comment (joepeeples) [inline docs/whats-new.asciidoc:57]: Nit: This describes expanding it just once, not expanding it and then expanding it again.
```suggestion
The redesigned alert details experience presents relevant context, insights, and more, while investigating an alert. Use the collapsed view to access summarized information, and then expand each section to open detailed views. Additional improvements include:
```

File: docs/whats-new.asciidoc [hunk 5]
```diff
@@ -4,131 +4,91 @@
...
 
 [float]
-=== Discover vulnerabilities with the new Cloud Native Vulnerability Management dashboard
+=== Organization-wide onboarding for cloud security posture management on AWS
 
-The {security-guide}/vuln-management-dashboard.html[Cloud Native Vulnerability Management (CNVM) dashboard] gives you an overview of vulnerabilities detected in your cloud infrastructure.
-
-[role="screenshot"]
-image::whats-new/images/8.9/CNVM-dashboard.png[CNVM dashboard]
+This release automates the onboarding of every account within an AWS Organization to cloud security posture management (CSPM) — including existing and new accounts. With {security-guide}/cspm-get-started.html#cspm-setup[AWS CloudFormation], onboarding takes just a few clicks. This helps you quickly get a comprehensive view of the security posture of all your AWS accounts — both current and future.
```
Comment (joepeeples) [inline docs/whats-new.asciidoc:87]: Suggested edit to mix up repetitive sentence structure:
```suggestion
This release automates the onboarding of every account within an AWS Organization to cloud security posture management (CSPM) — including existing and new accounts. With {security-guide}/cspm-get-started.html#cspm-setup[AWS CloudFormation], onboarding takes just a few clicks. This helps you quickly get a comprehensive view of the security posture of all your current and future AWS accounts.
```

**Review by joepeeples — COMMENTED** [review]
Looks great! Just added a few nit edits for consideration.

File: docs/whats-new.asciidoc [hunk 6]
```diff
@@ -4,131 +4,91 @@
...
+[role="screenshot"]
+image::whats-new/images/8.10/rules-coverage.png[MITRE ATT&CK® coverage page]
 
 [float]
-=== Interactive investigation guides are now generally available
-
-{security-guide}/interactive-investigation-guides.html[Interactive investigation guides], which suggest steps for triaging, analyzing, and responding to potential security issues, are now generally available. You can configure an interactive investigation guide when you create a new rule or edit an existing one.
+=== New prebuilt rule details flyout
 
+The new prebuilt rule details flyout allows you to examine the details of a prebuilt rule before you {security-guide}/prebuilt-rules-management.html[install or update] it. You can access this flyout by clicking a rule name on either the **Add Elastic Rules** page or the **Rule updates** table. The flyout displays the About, Definition, and Schedule sections, as shown on the rule details page. It also shows the setup and investigation guides for rules that have them.
```
Comment (jmikell821) [inline docs/whats-new.asciidoc:49]:
```suggestion
The new prebuilt rule details flyout allows you to examine the details of a prebuilt rule before you {security-guide}/prebuilt-rules-management.html[install or update] it. You can access this flyout by clicking a rule name on the **Add Elastic Rules** page or the **Rule updates** table. The flyout displays the **About**, **Definition**, and **Schedule** sections, as shown on the rule details page. It also shows the setup and investigation guides for rules that have them.
```

File: docs/whats-new.asciidoc [hunk 7]
```diff
@@ -4,131 +4,91 @@
...
 
 [float]
-=== Prebuilt rule updates
-
-Check out the {security-guide}/prebuilt-rules-downloadable-updates.html[latest updates] to prebuilt rules. To download the latest updates, refer to {security-guide}/prebuilt-rules-management.html#update-prebuilt-rules[Update Elastic prebuilt rules]. 
+=== Enhanced alert details flyout UI
 
-[float]
-=== Manage and filter alert tags
+The redesigned alert details experience presents relevant context, insights, and more, while investigating an alert. Use the collapsed view to access summarized information, and then expand each section to open detailed views. Additional improvements include:
```
Comment (jmikell821) [inline docs/whats-new.asciidoc:57]:
```suggestion
The redesigned alert details experience presents relevant context and insights while investigating an alert. Use the collapsed view to access summarized information, and then expand each section to open detailed views. Additional improvements include:
```

File: docs/whats-new.asciidoc [hunk 8]
```diff
@@ -4,131 +4,91 @@
...
 
 [float]
-=== Discover vulnerabilities with the new Cloud Native Vulnerability Management dashboard
+=== Organization-wide onboarding for cloud security posture management on AWS
 
-The {security-guide}/vuln-management-dashboard.html[Cloud Native Vulnerability Management (CNVM) dashboard] gives you an overview of vulnerabilities detected in your cloud infrastructure.
-
-[role="screenshot"]
-image::whats-new/images/8.9/CNVM-dashboard.png[CNVM dashboard]
+This release automates the onboarding of every account within an AWS Organization to cloud security posture management (CSPM) — including existing and new accounts. With {security-guide}/cspm-get-started.html#cspm-setup[AWS CloudFormation], onboarding takes just a few clicks. This helps you quickly get a comprehensive view of the security posture of all your current and future AWS accounts.
```
Comment (jmikell821) [inline docs/whats-new.asciidoc:87]:
```suggestion
This release automates the onboarding of every AWS Organization account to cloud security posture management (CSPM) — including existing and new accounts. With {security-guide}/cspm-get-started.html#cspm-setup[AWS CloudFormation], onboarding takes just a few clicks. This helps you quickly get a comprehensive view of the security posture of all your current and future AWS accounts.
```

File: docs/whats-new.asciidoc [hunk 9]
```diff
@@ -4,131 +4,91 @@
...
-The following enhancements have been added to Cases: 
-
-* You now have the option to {security-guide}/cases-open-manage.html#cases-ui-open[specify a category] for new and existing cases.
-* You can now {security-guide}/cases-open-manage.html#cases-lens-visualization[add Lens visualizations to cases] from anywhere within the {security-app}.
-* The case details *Alerts* tab now displays the number of alerts attached to a case.
-* Email notifications now follow a new and improved template.
-
+=== Cloud security posture management, now for Google Cloud
 
+{elastic-sec} 8.10 expands its cloud security posture management (CSPM) capabilities to cover {security-guide}/cspm-get-started-gcp.html[Google Cloud]. You can now assess and bolster the security posture of your GCP assets right from our platform.
```
Comment (jmikell821) [inline docs/whats-new.asciidoc:92]:
```suggestion
Cloud security posture management (CSPM) capabilities have been expanded to cover {security-guide}/cspm-get-started-gcp.html[Google Cloud]. You can now assess and bolster the security posture of your GCP assets right from our platform.
```

**Pull request merged**